controllers/_Settings.php

```diff
@@ -409,16 +378,17 @@ public static function setting_form_field( $name, $data ) {
 				$defaults = array(
 					'name' => $name,
 					'echo' => 1,
-					'show_option_none' => __( '-- Select --', 'sprout-invoices' ),
+					'show_option_none' => __( '-- Select --', 'help-scout' ),
 					'option_none_value' => '0',
 					'selected' => $data['default'],
 					);
 				$parsed_args = wp_parse_args( $data['args'], $defaults );
-				wp_dropdown_pages( $parsed_args ); ?>
-		<?php elseif ( $data['type'] == 'bypass' ) : ?>
-			<?php if ( isset( $data['output'] ) ) { echo $data['output']; } // not escaped ?>
+				wp_dropdown_pages( $parsed_args ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
If the whole point of this patch is to fix security issues, why are you ignoring this escaping?